.kokoro/release.cfg

```diff
@@ -4,3 +4,12 @@ env_vars: {
   key: "TRAMPOLINE_BUILD_FILE"
   value: ".kokoro/release.sh"
 }
+
+# Store the packages uploaded to rubygems.org, which
+# we can later use to generate SBOMs and attestations.
+action {
+ define_artifacts {
+   regex: "github/google-api-ruby-client/**/pkg/*.gem"
```

Comment:
Please use `github/google-api-ruby-client/*/pkg/*.gem` here so we explicitly exclude subdirectories of `generated`. Generated libraries are released by the separate `release-generated.cfg` job.

Speaking of which, we also need to add a similar section to that job's config, so that we capture those artifacts.